src/main/java/com/ericsson/ei/jsonmerge/MergeHandler.java

```diff
@@ -67,13 +67,15 @@ public String mergeObject(String id, String mergeId, RulesObject rules, String e
         try {
             // lock and get the AggregatedObject
             String aggregatedObject = getAggregatedObject(id, true);
+            LOGGER.debug("AGGREGATED OBJECT : " + aggregatedObject);
```

Comment:
Do you really want to log the entire aggregation twice? They can be very big, so it might clutter the log.

Reply:
removed logger